src/bin/deku_cli.ml

```diff
@@ -129,6 +129,20 @@ let ticket =
     Format.fprintf fmt "%S" (Tezos.Ticket_id.to_string ticket) in
   let open Arg in
   conv ~docv:"A ticket" (parser, printer)
+let vm_flavor =
+  let parser string =
+    (match string with
+    | "Lambda" -> Some `Lambda
```

Comment:
Naive question: Does it have to be polymorphic?
Why not defining a simple
```ocaml
type vm_type =
  | Lambda 
  | Dummy
```

Reply:
honest answer: I was too lazy to do that and we already use polymorphic variants freely in the cli